### DISCUSSION TITLE
4G & WiFi USB Adapter Compatibility

### DISCUSSION THREAD

**Discussion by tatoomana** [Ysurac/openmptcprouter#3710] [Q&A] [answered]
Hello,

First of all, I want to say that I am not a Linux expert and that the ARCH OS is unknown to me. Thanking you in advance for your indulgence.

I am currently preparing my shopping list to make a Live Streaming IRL setup.

To this end, I would like to know the list of USB Wifi and 4G adapters natively supported by OMR on Raspberry Pi.

Indeed, I want to buy a SIM7600G-H 4G dongle. But it is impossible to know if the device will work out-of-the-box. And if not, is there a tutorial to install the driver? (I do not know Arch)

Same for USB Wifi adapters. I connected a TP Link USB WiFi adapter, but it did not appear in the interfaces.

Thanking you in advance for your help.

![STREAM drawio](https://github.com/user-attachments/assets/c88f4968-c18b-4886-88a0-5aba3d12b34c)

**Comment by sieade245** [comment, marked as answer]
Unable to help with this but have you considered an ethernet based 4g access point instead? They are fairy cheap on AliExpress the performance isn't amazing but was faster than the usb ones I tested. Also you know it will work. Just need an ethernet hub to plug them all into. £15-20 each on Ali express

**Reply by tatoomana** [reply]
Hello, I have indeed opted for this solution. I have ordered portable 4G routers with RJ45 on battery. Thank you for your answer.
